src/main/java/io/datasearch/diseasedata/store/dengdipipeline/ingestion/data/DiseaseData.java

```diff
@@ -0,0 +1,63 @@
+package io.datasearch.denguestore.data;
+
+import org.geotools.data.Query;
+import org.geotools.filter.text.cql2.CQLException;
+import org.geotools.filter.text.ecql.ECQL;
+import org.opengis.feature.simple.SimpleFeature;
+import org.opengis.feature.simple.SimpleFeatureType;
+import org.opengis.filter.Filter;
+
+import java.util.List;
+
+/**
+ * Interface for Disease Data Features.
+ */
+public interface DiseaseData {
+
+    String getTypeName();
+    SimpleFeatureType getSimpleFeatureType();
+    List<SimpleFeature> getFeatureData();
+    List<Query> getTestQueries();
+    Filter getSubsetFilter();
```

Comment:
Don't need this. We only need to execute inserting operations. We are not going to use this model for querying